### PR TITLE
chore: make mutations distinct in npx pepr monitor

### LINKED ISSUE #560: make mutations distinct in npx pepr monitor
### Is your feature request related to a problem? Please describe.
It is easy to miss what Pepr has mutated in `npx pepr monitor`. We need the mutations to stand out more

### Describe the solution you'd like

- **Given** a mutation occurred and something is not right in the cluster
- **When** you issue `npx pepr monitor`, you see the mutations/validations distinctly 
- **Then** it dawns on you... it was a mutation that caused it that caused the behavior 

### Describe alternatives you've considered
(optional) A clear and concise description of any alternative solutions or features you've considered.

### PR DESCRIPTION
**Pull request by schaeferka**
## Description
Updated icons/emojis for mutations.

Screenshot 2024-02-23 at 02 29 56

...

## Related Issue

Fixes #560 

Relates to #

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Other (security config, docs update, etc)

## Checklist before merging

- [ ] Test, docs, adr added or updated as needed
- [ ] [Contributor Guide Steps](https://github.com/defenseunicorns/pepr/blob/main/CONTRIBUTING.md#submitting-a-pull-request) followed

### CODE REVIEW

File: src/cli/monitor.ts [hunk 1]
```diff
@@ -59,7 +59,7 @@ export default function (program: RootCmd) {
               const uid = payload.uid;
 
               if (isMutate) {
-                const allowOrDeny = payload.res.allowed ? "✅" : "❌";
+                const allowOrDeny = payload.res.allowed ? "🔀" : "❗";
```
Comment (cmwylie19) [inline src/cli/monitor.ts]:
```suggestion
                const allowOrDeny = payload.res.allowed ? "🔀" : "❌";
```

Comment (cmwylie19) [reply src/cli/monitor.ts]: Could be just me..

I'm of the opinion that the `not allowed` should be `X`. I read `!` (exclamation)  as attention, or warning. 

That said, i get why you did it, to make it more obvious what is mutate and validate.

Comment (cmwylie19) [reply src/cli/monitor.ts]: resolved syncro

File: package-lock.json [hunk 2]
```diff
@@ -42,7 +42,7 @@
         "@typescript-eslint/eslint-plugin": "6.15.0",
         "@typescript-eslint/parser": "6.15.0",
         "commander": "11.1.0",
-        "esbuild": "0.19.10",
+        "esbuild": "^0.19.10",
```
Comment (cmwylie19) [inline package-lock.json]:
```suggestion
        "esbuild": "0.19.10",
```

File: package.json [hunk 3]
```diff
@@ -58,7 +58,7 @@
     "@typescript-eslint/eslint-plugin": "6.15.0",
     "@typescript-eslint/parser": "6.15.0",
     "commander": "11.1.0",
-    "esbuild": "0.19.10",
+    "esbuild": "^0.19.10",
```
Comment (cmwylie19) [inline package.json]:
```suggestion
    "esbuild": "0.19.10",
```

File: package-lock.json [hunk 4]
```diff
@@ -7878,4 +7878,4 @@
       }
     }
   }
-}
+}
```
Comment (cmwylie19) [inline package-lock.json]:
```suggestion
}

```

Comment (cmwylie19) [reply package-lock.json]: add new line -> https://unix.stackexchange.com/questions/484802/should-the-last-line-in-a-configuration-file-have-a-newline-after-it

**Comment by schaeferka** [comment]
Updated mutate information display:

Screenshot 2024-02-26 at 23 02 33

File: src/cli/monitor.ts [hunk 5]
```diff
@@ -59,8 +59,16 @@ export default function (program: RootCmd) {
...
-                const allowOrDeny = payload.res.allowed ? "✅" : "❌";
+                const plainPatch = atob(payload.res.patch) || "";
+                const patch = JSON.stringify(JSON.parse(plainPatch), null, 2);
+
+                const patchType = payload.res.patchType || payload.res.warnings || "";
+
+                const allowOrDeny = payload.res.allowed ? "🔀" : "🚫";
                 console.log(`\n${allowOrDeny}  MUTATE     ${name} (${uid})`);
+                if (patchType.length > 0) {
+                  console.log(`\n\u001b[1;34m${patchType}\n${patch}\u001b[0m`);
```
Comment (cmwylie19) [inline src/cli/monitor.ts]:
```suggestion
                  console.log(`\n\u001b[1;34m${patch}\u001b[0m`);
```

**Review by cmwylie19 — APPROVED** [review]

**Pull request merged**